_data/tasks.yml

```diff
@@ -23,4 +23,6 @@ toc:
   section:
   - title: Assigning Pods to Nodes
     path: /docs/tasks/administer-cluster/assign-pods-nodes/
+  - title: Using DNS Horizontal Autoscaling
```

Comment:
Thanks for advising. Since above two tutorials seem not to be directly related to this topic, I prefer to keep it under Tasks/Administering. And may relocate it in the future if more tutorial sections come up.

Because we also put a pointer in the DNS page, it wouldn't be too hard to find.